docs/protocol-design/work.md

```diff
@@ -14,16 +14,51 @@ Existing content:
 
 ---
 
-## Spam resistance focus
+## Spam resistance
+
+!!! quote ""
+	Proof-of-Work (PoW) is used as a method to deter spam transactions on the network. It is **not** used in the consensus mechanism.
```

Comment:
Perhaps reference the Hashcash paper somewhere here, similar to original whitepaper: http://docs.nano.org/whitepaper/english/#fn:9